website/frontend/templates/docs/build-windows.html

```diff
@@ -6,113 +6,192 @@
+pip3 install Mutagen>=1.37
+pip3 install discid</pre>
+
+      <h2 id="python_picard1">Python for Picard version 1.4.x</h2>
+
+      <h3 id="python27">Python 2.7.x</h3>
 
       <ul>
-        <li> <a rel="nofollow" href="https://sites.google.com/site/pydatalog/python/pip-for-windows">https://sites.google.com/site/pydatalog/python/pip-for-windows</a>
-        </li>
-        <li>Download to a location you find again (no installation)
+        <li>
+          Download and install Python 2.7.x for Windows from 
+          <a rel="nofollow" target="_blank" href="http://python.org/downloads/">http://python.org/downloads/</a>
         </li>
-        <li>in Pip_Win:
-          <ul>
-            <li>
-              <code>pip install mutagen</code>for Mutagen (1.22 tested)
-            </li>
-            <li>
-              <code>pip install discid</code>for python-discid (1.1.x tested)
-            </li>
-          </ul>
+        <li>
+          Python should add itself to your %PATH% environment variable, 
+          but if not add both the Python install directory and its scripts subdirectory to %PATH%
         </li>
       </ul>
 
-      <h3 id="gettext">gettext</h3>
+      <h3 id="python2_packages">Python 2 packages</h3>
+      
+      <p>
+         You need need the following additional python packages to run Picard 1.4.x from source:
+      </p>
 
       <ul>
-        <li> <a rel="nofollow" href="http://gnuwin32.sourceforge.net/packages/gettext.htm">http://gnuwin32.sourceforge.net/packages/gettext.htm</a>
-        </li>
-        <li>Add the gettext folder to&#160;%PATH% (normally <i>C:\Programs\GnuWin32\bin</i>)
+        <li>PyQt 4.10 or later</li>
+        <li>Mutagen 1.37 or later</li>
+        <li>discid</li>
+      </ul>
+      
+      <p>
+        The easiest way to install them is using PIP. 
+        PIP is included as standard in Python 2.7.9 or later - so this is a good reason to upgrade if you
+        already have an earlier version of Python 2.7 installed.
+        Alternatively you can install PIP separately on earlier versions or install the above packages manually.
+      </p>
+        
+      <p>
+        Assuming that you want to use PIP, open an Administrator command prompt, navigate to the Picard source directory and type:
+      </p>
+      
+      <pre>pip install PyQt4>=4.10
+pip install Mutagen>=1.37
+pip install discid</pre>
+      
+      <p>
+        If you wish to install manually, here are the links you need:
+      </p>
+      
+      <ul>
+        <li>PyQt4 - <a rel="nofollow" target="_blank" href="https://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.4/">https://sourceforge.net/projects/pyqt/files/PyQt4/PyQt-4.11.4/</a></li>
+        <li>
+          discid - <a rel="nofollow" target="_blank" href="https://musicbrainz.org/doc/libdiscid">https://musicbrainz.org/doc/libdiscid</a><br />
+          Download "libdiscid-0.6.1-win32.zip" and put "discid.dll" in your Windows\System32 folder
```

Comment:
AFAIK PIP will install libdiscid as a dependency which includes the dll - but I am not able to test this.

(I am of the view that if someone wants to develop picard they should be capable of working out if the instructions are wrong and giving us the feedback. I wouldn't be nearly as blasé about this if it was a non-developer. I just don't have time or resources to build a clean PC to test this, and believe that this is better than having no instructions at all which is where I started.)